### PR TITLE
Updated deno to 1.7.0

### PR DESCRIPTION
**Pull request by lunaroyster**
Updates deno to the latest version.

### CODE REVIEW

File: languages/deno.toml [hunk 1]
```diff
@@ -6,7 +6,7 @@ extensions = [
 packages = [
 ]
 setup = [
-  "curl -fsSL https://deno.land/x/install/install.sh | DENO_INSTALL=/usr sh -s v1.1.0",
+  "curl -fsSL https://deno.land/x/install/install.sh | DENO_INSTALL=/usr sh -s v1.6.2",
```
Comment (satyarohith) [inline languages/deno.toml:9]:
```suggestion
  "curl -fsSL https://deno.land/x/install/install.sh | DENO_INSTALL=/usr sh -s v1.6.3",
```

File: languages/deno.toml [hunk 2]
```diff
@@ -6,7 +6,7 @@ extensions = [
...
   "npm i -g typescript-language-server@0.4.0 typescript@3.8.3"
```
Comment (satyarohith) [inline languages/deno.toml:10]:
```suggestion
  "npm i -g typescript-language-server@0.4.0 typescript@4.1.3"
```

Comment (satyarohith) [reply languages/deno.toml:10]: This is the version running on Deno `1.6.3`.

**Review by satyarohith — CHANGES_REQUESTED** [review]

**Comment by Liamolucko** [comment]
`deno lsp` was added in 1.6, it should probably be used instead of `typescript-language-server`.

**Review by cbrewster — APPROVED** [review]
⬆️ Have you tested this locally to make sure LSP still works?

**Comment by lunaroyster** [comment]
Awesome, LSP's working now.

**Pull request merged**